idx/open-in-idx-template/idx-template.json

```diff
@@ -0,0 +1,35 @@
+{
+    "name": "Open in IDX for Maps SDK for Android Samples",
+    "description": "This template allows opening Android tutorials in IDX IDE",
+    "categories": ["Backend"],
```

Comment:
Delete, since this is not being shown in the template gallery and it's not a backend template (leftover from cloning the Python template)

Reply:
Done. Consulted with my POC from IDX team. They recommended "Mobile" category.